types/util.promisify/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for util.promisify 1.0
+// Project: https://github.com/ljharb/util.promisify#readme
+// Definitions by: Adam Voss <https://github.com/adamvoss>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ Note that ES6 modules cannot directly export callable functions.
+ *~ This file should be imported using the CommonJS-style:
+ *~   import x = require('util.promisify');
+ *~
+ *~ Refer to the documentation to understand common
+ *~ workarounds for this limitation of ES6 modules.
+ */
+export = promifify;
+
+// tslint:disable-next-line ban-types
+declare function promifify(f: Function): Function;
```

Comment:
It is the same as the [node` util.promisify`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/index.d.ts#L4013-L4016).  My TypeScript-fu is weak, but based on your suggestions showing variadic parameters, I think I found the solution.  If it is accepted here, then I can open another PR against the `node` definition file.